dspace/config/launcher.xml

```diff
@@ -366,4 +366,11 @@
             <class>org.dspace.statistics.AnonymizeStatistics</class>
         </step>
     </command>
+    <command>
+        <name>canvas-dimensions</name>
+        <description>Add canvas width and height metadata for IIIF enabled items.</description>
```

Comment:
We may want to improve this description, as I first understood it to say that it adds the canvas height/width __to the Item__. But, it's obviously adding it to the Bitstream.  Maybe say something like this?

"Automatically add the IIIF canvas width/height metadata to Bitstreams of IIIF enabled Items".